History.md

```diff
@@ -1,5 +1,10 @@
 ## v.NEXT
 
+* `observe`/`observeChanges` callbacks are now bound.
```

Comment:
What if you're not using `observe`/`observeChanges` from within a `publication`?